src/renderer/Generics/redesign/HeaderButton/headerButton.css.ts

```diff
@@ -13,6 +13,7 @@ export const container = style({
   width: 28,
   height: 28,
   borderRadius: 5,
+  zIndex: 9999999,
```

Comment:
So the reason for this was that the draggable window edge on the top has a height given by danny that goes below the buttons and it was making it impossible to click, so I added a high z-index (although, it could definitely be lower haha)